utils/spacewalk-common-channels.ini

```diff
@@ -3020,3 +3020,50 @@ gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni
 gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/EL8-Uyuni-Client-Tools/EL_8/
+
+[debian-11-pool-amd64-uyuni]
```

Comment:
Can you move the Debian 11 repos to be below those for Debian 10? That way it will be easier to maintain :-)